operator/v1/types_network.go

```diff
@@ -425,6 +425,88 @@ type OVNKubernetesConfig struct {
 	// egressIPConfig holds the configuration for EgressIP options.
 	// +optional
 	EgressIPConfig EgressIPConfig `json:"egressIPConfig,omitempty"`
+	// ipv4 allows users to configure IP settings for IPv4 connections. When ommitted,
+	// this means no opinions and the default configuration is used. Check individual
+	// fields within ipv4 for details of default values.
+	// +optional
+	IPv4 *IPv4OVNKubernetesConfig `json:"ipv4,omitempty"`
+	// ipv6 allows users to configure IP settings for IPv6 connections. When ommitted,
+	// this means no opinions and the default configuration is used. Check individual
+	// fields within ipv4 for details of default values.
+	// +optional
+	IPv6 *IPv6OVNKubernetesConfig `json:"ipv6,omitempty"`
+}
+
+type IPv4OVNKubernetesConfig struct {
+	// internalTransitSwitchSubnet is a v4 subnet in IPV4 CIDR format used internally
+	// by OVN-Kubernetes for the distributed transit switch in the OVN Interconnect
+	// architecture that connects the cluster routers on each node together to enable
+	// east west traffic. The subnet chosen should not overlap with other networks
+	// specified for OVN-Kubernetes as well as other networks used on the host.
+	// The value cannot be changed after installation.
```

Comment:
Can you go from no opinion to a value?  Can you add CEL to ensure we don't allow this change?

Reply:
are you asking if we can change the statement from "no opinion" to what the current default is? also we currently dont allow the value to be changed after install but are currently working on making it so that is possible so how do you want to approach that, i can add some CEL to make sure it cant be changed for now but it will just be removed when that merges